projects/storefrontapp-e2e-cypress/cypress/helpers/my-company/models/MyCompanyRowConfig.ts

```diff
@@ -0,0 +1,51 @@
+export interface MyCompanyRowConfig {
```

Comment:
Question: For extensibility reasons, maybe all config elements should be optional?